Sources/Views/AvatarView.swift

```diff
@@ -89,7 +89,7 @@ open class AvatarView: UIView {
         if width == 0 || height == 0 {return UIImage()}
         var font = placeholderFont
 
-        _ = UIGraphicsBeginImageContext(CGSize(width: width, height: height))
+      _ = UIGraphicsBeginImageContextWithOptions(CGSize(width: width, height: height), false, UIScreen.main.scale)
```

Comment:
Doesn't look like we ever end this image context 🤔 Worth chucking it in a defer?

Reply:
Yep, good point 👍